practice-one/exercise-2/part-1/js/script.js

```diff
@@ -0,0 +1,9 @@
+const getInfo = () => {
```

Comment:
Why did you set function name as `getInfo`? You just styled for the paragraph text only.